packages/sdk-react/src/components/counterparts/CounterpartDetails/CounterpartView/CounterpartView.tsx

```diff
@@ -104,6 +104,16 @@ export const CounterpartView = (props: CounterpartViewProps) => {
     deleteCounterpart(handleCloseDeleteCounterpartDialog);
   }, [deleteCounterpart, handleCloseDeleteCounterpartDialog]);
 
+  const getDefaultOrganizationContact = useCallback(() => {
+    if (counterpart && isOrganizationCounterpart(counterpart)) {
+      const matchingContact = contacts?.find(
+        (contact) => contact.email === counterpart.organization?.email
+      );
+
+      return matchingContact?.is_default;
+    }
+  }, [counterpart, contacts]);
+
```

Comment:
But why we removed here useCallback?

Reply:
What are the benefits? The ternary is very lightweight and iteration over just a few items it's not a heavy operation.

Reply:
I notice that we use useCallback in many places within our app, even for simple cases. I’m trying to understand the reasoning behind this.

<img width="536" alt="Screenshot 2024-08-08 at 02 06 06" src="https://github.com/user-attachments/assets/fb298d55-90a8-4389-acdf-a953ed85e570">

Are we relying or putting bet on the React compiler and auto-memoization in the future? Currently, we use it even for lightweight iterations and setters for useState. Can you explain this approach, or suggest an ESLint rule to avoid using useCallback for iterations?

Reply:
I think we explained that there are many incorrect uses of `useCallback` and `useMemo` in our React codebase. You should not copy incorrect patterns.